ReadMe.md

```diff
@@ -166,6 +166,7 @@ Front-end | フロントエンド | |
 Management | 管理 | かんり | Of people, software, etc
 Manager | 管理者 | かんりしゃ |
 Infrastructure | インフラ | |
+Continuous integration | 継続的インテグレーション | けいぞくてきいんてぐれーしょん |
```

Comment:
Nop, it sounds good to me and it's probably more readable too :) I will updated.